src/common/readfield_nc.f90

```diff
@@ -855,6 +1034,52 @@ subroutine nfcheckload2d(ncid, varname, start, length, field, return_status)
   end if
 end subroutine nfcheckload2d
 
+subroutine nfcheckload2d_r64(ncid, varname, start, length, field, return_status)
+  use ieee_arithmetic, only: ieee_value, IEEE_QUIET_NAN
```

Comment:
This function is never used, and it usually doesn't make any sense to store meteorology or other input data in 64bit.